src/release.json

```diff
@@ -0,0 +1,302 @@
+[
+  {
+    "name": "v0.25.0",
```

Comment:
Can you describe how this file was produced

It's better to have a documentation

Reply:
https://github.com/zhmushan/setup-deno/blob/refactor/src/installer.ts#L125
I think we should fetch `release.json` from web finally. If not, once deno updated, we need to update too

Reply:
Yeah.

that is next step. We should put this file into `/docs` folder and enable `gh-pages`

When this file is finalized, another PR can be opened

add files:

-/docs/release.json
-/docs/README.md

then load release.json file with http in this PR

Reply:
could you open another PR for this?

after the `release.json` file is merged into master

then consider this PR again

Reply:
Maybe we can use `now.sh` to deploy `git page` to forward data from `GitHub API`. This way we can avoid having to update release.json every time

Reply:
Even if you deploy with now, you still need to update release.json

I don’t think it is necessary to rely on third-party services

`gh-pages` is enough

Reply:
#5

Reply:
done.